serverless/develop/serverless-using-brokers.adoc

```diff
@@ -6,16 +6,25 @@ include::modules/serverless-document-attributes.adoc[]
 
 toc::[]
 
-Brokers can be used in combination with xref:../../serverless/knative_eventing/serverless-triggers.adoc#serverless-triggers[triggers] to deliver events from an xref:../../serverless/discover/knative-event-sources.adoc#knative-event-sources[event source] to an event sink.
+Brokers can be used in combination with xref:../../serverless/develop/serverless-triggers.adoc#serverless-triggers[triggers] to deliver events from an xref:../../serverless/discover/knative-event-sources.adoc#knative-event-sources[event source] to an event sink.
 
 image::serverless-event-broker-workflow.png[Broker event delivery overview]
 
-Events can be sent from an event source to a broker as an HTTP POST request.
+Events can be sent from an event source to a broker as an HTTP POST request. After events have entered the broker, they can be filtered by https://github.com/cloudevents/spec/blob/v1.0/spec.md#context-attributes[CloudEvent attributes] using triggers, and sent as an HTTP POST request to an event sink.
```

Comment:
I can't find definite confirmation of this in the IBM style guide, but my understanding is that e.g. `POST` and `PUT`  are commands.
```suggestion
Events can be sent from an event source to a broker as an HTTP `POST` request. After events have entered the broker, they can be filtered by https://github.com/cloudevents/spec/blob/v1.0/spec.md#context-attributes[CloudEvent attributes] using triggers, and sent as an HTTP `POST` request to an event sink.
```

Reply:
+1 I like this suggestion